outlook/src/classes/Company.ts

```diff
@@ -1,4 +1,14 @@
 import Address from './Address';
+
+/***
+ * id reserved for companies which we fetch directly from IAP
+ */
+const ID_COMPANY_FROM_REVEAL: number = -2;
+/***
+ * id reserved for empty companies
+ */
+const ID_EMPTY_COMPANY: number = -1;
```

Comment:
ID_COMPANY_EMPTY to be coherent with ID_COMPANY_FROM_REVEAL ?

Reply:
Yes!